install_config/aggregate_logging.adoc

```diff
@@ -309,45 +336,30 @@ $ oc get dc --selector logging-infra=elasticsearch
 ----
 ====
 
-////
-To scale Elasticsearch deployments, create and add more deployments, being aware
-of the cluster parameter restrictions. The deployer uses a template to create
-Elasticsearch deployments. These deployments will be named differently, but will
-all have the 'service/logging-es-cluster' prefix:
-
-====
-----
-$ oc process logging-es-template | oc create -f -
-----
-====
-////
-
 *Persistent Elasticsearch Storage*
 
-The deployer creates an ephemeral deployment in which all of a pod's data is
-lost upon restart. For production, persistent storage is recommended. The
-following example specifies a persistent storage volume for the Elasticsearch
-deployment. You can use the `oc volume` command to add a created volume to a
-deployment:
+The deployer creates an ephemeral deployment in which all of a pod's
+data is lost upon restart. For production usage, persistent storage is
+required. You can use the `oc volume` command to add a created volume
+to each deployment. The following example specifies a volume for an
+Elasticsearch instance (using a
+link:../architecture/additional_concepts/storage.html#persistent-volume-claims[PersistentVolumeClaim]):
 
 ====
 ----
 $ oc volume dc/logging-es-rca2m9u8 \
           --add --overwrite --name=elasticsearch-storage \
-          --type=hostPath --path=/path/to/storage
+          --type=persistentVolumeClaim --claim-name=logging-es-1
 ----
 ====
 
 [NOTE]
 ====
-Allowing the pods to mount host volumes as above usually requires adding the
-`aggregated-logging-elasticsearch` service account to the privileged SCC,
-similarly as for Fluentd above.
+Any available volume type can be used, such as a host-mount, but the
+recommended volume type is a PersistentVolumeClaim.
 ====
 
-You can use any volume type, such as
-link:../install_config/persistent_storage/persistent_storage_nfs.html[NFS].
-
+ifdef::openshift-enterprise[]
```

Comment:
in origin this whole section is superseded by making parameters available to the deployer.